prisma/seed.js

```diff
@@ -0,0 +1,28 @@
+import { dataSet } from "../data/data.js";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+const dataAgency = dataSet.agencies.map((agency) => ({
+  name: agency.name,
+  city: agency.city,
+  region: agency.region,
+  size: agency.companySize,
+  website: agency.website,
+  logo: agency.eguideImageSrc,
+}));
+
+async function main() {
+  for (let agency of dataAgency) {
+    await prisma.agencies.create({
```

Comment:
This works! Nice that you wrote a seed script: 

2 thoughts

1. Maybe we should filter the data for unwanted record (no website for instance) 
2. This script will seed by doing 1 db call for each agency. It will be more efficient to use: 
prisma.agencies.createMany({})
https://www.prisma.io/docs/concepts/components/prisma-client/crud#create-multiple-records